ethereum/referencetests/src/reference-test/java/org/hyperledger/besu/ethereum/vm/GeneralStateReferenceTestTools.java

```diff
@@ -206,4 +227,33 @@ public static void executeTest(final GeneralStateTestCaseEipSpec spec) {
   private static boolean shouldClearEmptyAccounts(final String eip) {
     return !SPECS_PRIOR_TO_DELETING_EMPTY_ACCOUNTS.contains(eip);
   }
+
+  private static void logWorldState(final ReferenceTestWorldState worldState) {
+    ObjectMapper mapper = new ObjectMapper();
+    ObjectNode worldStateJson = mapper.createObjectNode();
+    worldState.streamAccounts(Bytes32.ZERO, Integer.MAX_VALUE)
+        .forEach(
+            account -> {
+              ObjectNode accountJson = mapper.createObjectNode();
+              accountJson.put("nonce", Bytes.ofUnsignedLong(account.getNonce()).toShortHexString());
+              accountJson.put("balance", account.getBalance().toShortHexString());
+              accountJson.put("code", account.getCode().toHexString());
+              ObjectNode storageJson = mapper.createObjectNode();
+              var storageEntries = account.storageEntriesFrom(Bytes32.ZERO, Integer.MAX_VALUE);
+              storageEntries.values().stream()
+                  .map(
+                      e ->
+                          Map.entry(
+                              e.getKey().orElse(UInt256.ZERO),
```

Comment:
displaying zero is not true, I will prefer return Bytes.EMPTY in this case

Reply:
Fine by me. Note, ZERO was somewhat following suit with https://github.com/hyperledger/besu/blob/ff5266af9ea4fc578ffeaacdee948435af4dbbeb/ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/EvmToolCommand.java#L581